locations/spiders/mma_fr.py

```
@@ -0,0 +1,30 @@
+    wanted_types = ["InsuranceAgency"]
+    drop_attributes = ["facebook", "image"]
+
+    def post_process_item(self, item, response, ld_data, **kwargs):
+        item["branch"] = (item.pop("name", "") or "").removeprefix("AGENCE D'ASSURANCE MMA ")
+        item["opening_hours"] = self.parse_opening_hours(ld_data)
+        apply_category(Categories.OFFICE_INSURANCE, item)
+        yield item
+
+    def parse_opening_hours(self, ld_data: dict) -> OpeningHours:
+        oh = OpeningHours()
+        for l in ld_data["openingHours"]:
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)